### PR TITLE
YOGA-4 Add images to home page

### PR DESCRIPTION
**Pull request by angeliquerivera**

### CODE REVIEW

**Comment by chrismejia** [comment]
@angeliquerivera please add `.next` to `.gitignore` file, then commit/push.

The `.next` directory contains build artifacts, cached files, and compiled output, which are all generated automatically when you build or start your Next.js application. Since these files are environment-specific and can be regenerated, they don’t need to be version-controlled.

Will review after change made.

File: app/layout.tsx [hunk 1]
```diff
@@ -10,8 +10,8 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <Navbar />
       <body>{children}</body>
+      <Navbar />
```
Comment (chrismejia) [inline app/layout.tsx:14]: Just confirming here, guessing your intention is to have a layout where Navbar is sticky to the bottom of the page?

Comment (angeliquerivera) [reply app/layout.tsx]: Switched Navbar component position

**Review by chrismejia — APPROVED** [review]
Deploy preview LGTM 👍🏼

**Pull request merged**